export class NotFoundException extends Error {
  name = 'NOT_FOUND_EXCEPTION';

  constructor(message) {
    const _message = message ?? 'NÃ£o encontrado';
    super(_message);
  }
}
